test/Altinn.Platform.Authentication.Tests/Controllers/LogoutControllerTests.cs

```diff
@@ -265,6 +266,49 @@ public async Task Logout_LogedIn_RedirectToIss()
             }
         }
 
+        [Fact]
+        public async Task Logout_HandleLoggedOut_RedirectToRedirectUrl()
+        {
+            // Arrange
+            HttpClient client = CreateClient();
+
+            HttpRequestMessage requestMessage = new HttpRequestMessage(HttpMethod.Get, "/authentication/api/v1/handleloggedout");
+            requestMessage.Headers.Add("Cookie", "AltinnLogoutInfo" + "=" + "SystemuserRequestId=c0970300-005c-4784-aea6-5e7bac61b9b1");
+
+            // Act
+            HttpResponseMessage response = await client.SendAsync(requestMessage);
+
+            // Assert
+            Assert.Equal(System.Net.HttpStatusCode.Found, response.StatusCode);
+
+            IEnumerable<string> values;
+            if (response.Headers.TryGetValues("location", out values))
+            {
+                Assert.Equal("https://smartcloudaltinn.azurewebsites.net/", values.First());
```

Comment:
Sjekk at cookie er slettet også

Reply:
Lagt på sjekk for cookie i begge testene